.github/workflows/typedoc-generate-gitbook-docs.yml

```diff
@@ -0,0 +1,54 @@
+name: Deploy TypeDoc to GitBook
+
+on:
+  push:
+    branches:
+      - develop
+  
+env:
+  NODE_VERSION: 18.x
+  ENTRY_FILE: "packages"
+  CONFIG_PATH: "tsconfig.base.json"
+  USES_PNPM: "true"
+  
+jobs:
+  deploy:
+    concurrency: ci-${{ github.ref }}
+    runs-on: ubuntu-latest
+    if: "!contains(github.event.head_commit.message, '[skip ci]')"
+    permissions:
+      contents: write
+    steps:
+      - uses: actions/checkout@v4
+  
+      - uses: actions/setup-node@v3
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+  
+      - name: Setup pnpm
+        uses: pnpm/action-setup@v4
+        with:
+          version: 9
+  
+      - name: Install dependencies
+        run: pnpm install --no-frozen-lockfile
+    
+      - name: Build project
+        run: pnpm build
+    
+      - name: Build documentation
+        run: pnpm docs:generate-markdown
+    
+      - name: Deploy to GitBook
+        env:
+          GITBOOK_TOKEN: ${{ secrets.GITBOOK_TOKEN }}
+          GH_PAT: ${{ secrets.GH_PAT }}
+        run: |
+          git clone --branch=git-book https://near:${{ secrets.GH_PAT }}@github.com/near/near-sdk-js.git book
+          rsync -av --delete markdown-docs/ book/markdown-docs/
+          cd book
+          git config user.name "GitHub Actions"
+          git config user.email "actions@github.com"
+          git add markdown-docs/
+          git commit -m "Update documentation [skip ci]"
+          git push https://near:${{ secrets.GH_PAT }}@github.com/near/near-sdk-js.git git-book
```

Comment:
There is no `GH_PAT` secret. As I mentioned in DM, there is `MY_GITHUB_TOKEN` secret associated with my `frol` account, so not `near:`.